unbound-install.sh

```diff
@@ -83,7 +83,7 @@ if [[ "$OS" = "arch" ]]; then
   pacman -Syu unbound
 
   # Get root servers list
-  curl -o /etc/unbound/root.hints https://www.internic.net/domain/named.cache
+	wget -O /etc/unbound/root.hints https://www.internic.net/domain/named.cache
```

Comment:
Please fix the indentation

Reply:
deleted this branch a while ago, sorry cant make anymore changes to this file.

if you want i can make a new pull.

Reply:
As you want :)